aioesphomeapi/client.py

```diff
@@ -1286,6 +1289,13 @@ def subscribe_voice_assistant(
             ]
             | None
         ) = None,
+        handle_announce_finished: (
```

Comment:
Wouldn't it be better to have this callback in the `send_voice_assistant_announce` method?

Reply:
That should work fine, and would make the HA side simpler.

Reply:
Moved the wait inside aioesphomeapi